azureiot.sln

```diff
@@ -95,6 +95,8 @@ Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "Solution Items", "Solution
 EndProject
```

Comment:
Same comment here as weel. Can we have a readme defining how to use this sample in conjunction with device sample.

Reply:
That's a good point. Sure I'll add a task to do this.